addon/utils/pretty-percentage.js

```diff
@@ -0,0 +1,10 @@
+export default function prettyPercentage(decimalValue, precision=0) {
```

Comment:
It'd also be nice to see this use the existing `roundNumber` util instead of reimplementing it. Actually, if you did that, this is for sure a helper.